routing/endpointregistry_test.go

```diff
@@ -114,6 +114,51 @@ func TestDoRemovesOldEntries(t *testing.T) {
 	assert.Equal(t, int64(0), mRemoved.InflightRequests())
 }
 
+func TestEndpointRegistryPostProcessor(t *testing.T) {
+	beginTestTs := time.Now()
+	r := routing.NewEndpointRegistry(routing.RegistryOptions{})
+
+	routing.SetNow(r, func() time.Time {
+		return beginTestTs
+	})
+	shuntRoute := &routing.Route{
+		Route: eskip.Route{
+			BackendType: eskip.ShuntBackend,
+		},
+	}
+	loopRoute := &routing.Route{
+		Route: eskip.Route{
+			BackendType: eskip.LoopBackend,
+		},
+	}
+	dynamicBackendRoute := &routing.Route{
+		Route: eskip.Route{
+			BackendType: eskip.DynamicBackend,
+		},
+	}
+	lbBackendRoute := &routing.Route{
+		Route: eskip.Route{
+			BackendType: eskip.LBBackend,
+		},
+		LBEndpoints: []routing.LBEndpoint{
+			{Host: "endpoint1.test:80", Metrics: r.GetMetrics("endpoint1.test:80")},
+			{Host: "endpoint2.test:80", Metrics: r.GetMetrics("endpoint2.test:80")},
+		},
+	}
+	networkBackendRoute := &routing.Route{
+		Route: eskip.Route{
+			BackendType: eskip.NetworkBackend,
+		},
+		Host: "endpoint3.test:80",
+	}
+	r.Do([]*routing.Route{shuntRoute, loopRoute, dynamicBackendRoute, lbBackendRoute, networkBackendRoute})
```

Comment:
Maybe it would be cleaner to define locals variable 
```go
routes := []*routing.Route{
	&routing.Route{
		Route: eskip.Route{
			BackendType: eskip.ShuntBackend,
		},
	},
	&routing.Route{
		Route: eskip.Route{
			BackendType: eskip.LoopBackend,
		},
	}, 
	...
}
r.Do(routes}
```
This way you do not need to invent a lot of local var names for single use